libraries/AP_Motors/AP_MotorsCoax.h

```diff
@@ -51,6 +48,9 @@ class AP_MotorsCoax : public AP_MotorsMulticopter {
     // output - sends commands to the motors
     void                output_armed_stabilizing() override;
 
+    // set update rate to motors - a value in hertz
+    void                set_update_rate( uint16_t speed_hz ) override;
```

Comment:
nitpick: let's get rid of the extra spaces within the brackets..

Reply:
This is just a copy and paste from line 37-39. The function is moved from public to protected. 

Can remove the spaces, but renaming would make this diff bigger again.